Makefile

```diff
@@ -144,6 +144,7 @@ tools:
 	@go get -u -v golang.org/x/lint/golint
 	@go get -u -v github.com/VojtechVitek/rerun/cmd/rerun
 	@python3 -m pip install black
+	@if [[ "$$OSTYPE" == "darwin"* ]]; then brew install parallel; else sudo apt-get install -y parallel; fi
```

Comment:
Shouldn't `$$OSTYPE` actually be `$OSTYPE`?

And 2nd, what do you think of using an `elif` instead of `else` to check if it's a Linux machine? Not necessarily required, but good to have I think.

Reply:
For some reason, in makefiles `$$` gets converted to just `$`

I added the `elif`

Reply:
Got it. That's weird indeed.